src/app/hs-0/page.tsx

```diff
@@ -38,14 +40,16 @@ const members = [
   },
 ];
 
+// The only code that should be changed below this line is on line 51.
 export default function HS0() {
   return (
     <div className="bg-[#ECF0F1]">
```

Comment:
There are color constants inside tailwind for all the colors. Try `bg-oasis` and then do `ctrl`+`space` to get Intellisense with all the colors autocompleted for choices.

Reply:
Fixed!